### ISSUE TITLE
not model files model.tflite and incomplete training file with no dataset

### ISSUE THREAD

**Issue by Idanish123** [DEV-D-GR8/SignSense#3]
Encountering same missing files issues 

Missing Components
Model File:

model.tflite is required but not included
This is the trained transformer model
Training Data:

train.csv is referenced but not included
Contains sign labels and mappings
Complete Training Code:

ASL_Model_Training.ipynb is incomplete
Missing model architecture and training loop

**Comment by DEV-D-GR8** [comment]
1. Trained model file won't be provided because I have plans with this model regarding some mobile applications and I don't plan to open source it yet (maybe in future). Training code is given, you can train the model yourself.
2. Dataset can be downloaded from kaggle. It is clearly mentioned that the dataset used is GISLR.
3. Model architecture and loop is present. Vibe coding might not help you, try reading each line and you'll know.
